frontend/src/modules/Core/Constants/Routes.ts

```diff
@@ -7,6 +7,7 @@ export const DASHBOARD_PATH = '/dashboard'
 export const METRICS_PATH = '/metrics'
 export const EMAIL_PATH = '/email'
 export const TEMPLATE_EDITOR_PATH = '/templates'
+export const SETTINGS_PATH = '/dashboard/settings'
```

Comment:
Since there haven't been any nested paths, I think we can keep the trend of just using `/settings` instead of having it nested in dashboard